### PR TITLE
Nightmare biolab

### PR DESCRIPTION
**Pull request by Atemo**
* **Addressed Issue(s)**: #1303

* **Server Mode**: renewal

* **Description of Pull Request**: 
Entrance and exchange NPCs of nightmare biolab ep

- [x] Eventually implement the sc_ buffs given in the dungeon
- [ ] Update mob datas
- [ ] Code review

### CODE REVIEW

**Comment by ecdarreola** [comment]
Tested the PR and found that Item 22679 isn't consumed after opening it.

**Comment by Haikenz** [comment]
Amazing  <3 Just missing the diff item_db sql

**Comment by Atemo** [comment]
sql item_db will be automatically updated by rathena API when this branch will be merged to master. Less work for me !

**Comment by Haikenz** [comment]
I noticed that you do not have the mobs and skill in db

```
Seyren Windsor:  Ignition Break, Dragon Breath(Fire), Phantom Thrust( If  receive Long Ranged Attack), Spear Boomerang, Critical Wounds, Earthquake, Max Payne (Rude Attack)

Randel Lawrence: Genesis Ray, Holy Cross, Grand Cross, Dark Cross,  Smite, Guard, Defending Aura, Spear Boomerang

Kathryne Keyron: Tetra Vortex, Earth Strain, Comet, Waterball, Quagmire, Jupitel Thunder, Sight Thrasher, Fire Wall, Frost Driver, Safety Wall, Sight, Dark Strike

Celia Alde: Killing Cloud, Psychic Wave, Dispell, Fiberlock(Wide, like Brinaranea's), Safety Wall, Stone Curse, Thunderstorm, Sight, Mental Breaker.
 
Cecil Damon: Arrow Storm, Focused Arrow Strike, Double Strafe, Critical Wounds, Bleeding, Curse
 
Alphocchio Basil: Reverberation, Improvised Song(Waterball, Storm Gust(0 Damage) , Lord of Vermillion, Impressive Riff, Pulse Strike
 
Trentini: Improvised Song (Meteor Storm), Reverberation, Unchained Serenade, Earthquake, Wide Stun
 
Howard Alt-Eisen: Lava Flow, Maximise Power, Divest Armor, Divest Helm, Mammonite, Earthquake
 
Flamel Emure:  Mandragora Howling, Acid Bomb, Acid Terror, Bomb, Aid Potion, Max Payne, Increase Agi
 
Eremes Guile: Venom Impression, Meteor Assault, Soul Destroyer,  Venom Dust, Sonic Blow, Grimtooth, Max Payne(30% Hp or lower)
 
Gertie Wie: Shadow Spell (Meteor Storm), Full Divest,  Meteor Assault, Stalk, Hiding, Wide Confusion, Poison Mist (The Pink quagmire-like things from Faceworm Nest)
 
Margaretha Sorin: Wide Lex Aeterna, Silentium, Safety Wall, Lex Aeterna, Assumptio, Increase AGI, Decrease AGI, Holy Light, Pneuma
 
Chen Liu: Cursed Circle, Guillotine Fist, Change Undead, Critical Wounds
```

**Comment by crazystorm2017** [comment]
If you're fine with jRO skills, I can add it.

**Comment by Atemo** [comment]
Thanks, but we have the datas

**Comment by bruno199** [comment]
Sorry my bad english but i have few infos

Hello rAthena Community, ty for work in biolab nightamare but i have find erros

Basic Error
1: Normal monster are dropping "MVP card"
check:
Monster 3208 is normal eremes but drop mvp card "Guillotine Cross Eremes" id 4674 but correct drop is 4684 "True Eremes Guile" card
this error happened in all normal monsters 
3208.3209,3210,3211,3226,3227,3228,3229,3230,3231,3232
and need corrections

*Accord iro info and playing in map

2 HP monster are wrong
i'm play in "IRO" and farm in bio 5  2-3x for every week

in "rathena" hp normal mobs are low 
check this:
rathena database: 3208 eremes guile : 700000
iro monster: 3208 eremes guile 2380000

check my topic:
https://rathena.org/board/topic/115508-how-to-add-skills-in-bio-5-monster/

i'm send info in IRO monster:
bio5

if you want info for mvp HP check in kafra.kr
http://kafra.kr/#!/ko/KRO/mobdetail/3220
http://kafra.kr/#!/ko/KRO/mobdetail/3221
http://kafra.kr/#!/ko/KRO/mobdetail/3222
http://kafra.kr/#!/ko/KRO/mobdetail/3223
http://kafra.kr/#!/ko/KRO/mobdetail/3224
http://kafra.kr/#!/ko/KRO/mobdetail/3225
http://kafra.kr/#!/ko/KRO/mobdetail/3240
http://kafra.kr/#!/ko/KRO/mobdetail/3241
http://kafra.kr/#!/ko/KRO/mobdetail/3242
http://kafra.kr/#!/ko/KRO/mobdetail/3243
http://kafra.kr/#!/ko/KRO/mobdetail/3244
http://kafra.kr/#!/ko/KRO/mobdetail/3245
http://kafra.kr/#!/ko/KRO/mobdetail/3246

i don't have info for Miniboss HP and Status i need check this

3: normal monster have chance drop 2 Cursed Fragment and no drop Death Bin, this drop is only mvp or trading 3 soul each job for 1 death bin box
check this note
http://ziu-ro.blogspot.com/2017/02/sepulchre-of-victims-sakray-server.html

**Comment by Atemo** [comment]
@bruno199 
About 3/

- the chance to drop 1 Cursed Fragment is 100% ?
- 'normal' boss **never** drop Death Bin ?

File: npc/re/mobs/dungeons/lhz_dun_n.txt [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+			switch(rand(1,6)) {	// unknown coordinates
+				case 1:  .@x = 140;  .@y = 232;  break;
+				case 2:  .@x = 75;   .@y = 138;  break;
+				case 3:  .@x = 140;  .@y = 87;   break;
+				case 4:  .@x = 205;  .@y = 140;  break;
+				case 5:  .@x = 123;  .@y = 137;  break;
+				case 6:  .@x = 175;  .@y = 137;  break;
+			}
+			setarray .@mob_list[0], 3220,3221,3222,3223,3224,3225, 3240,3241,3242,3243,3244,3245,3246;
+			monster "lhz_dun_n",.@x,.@y,"--en--",.@mob_list[ rand(13) ],1, "lhz_dun_n::OnMyMVPDead";
```
Comment (ecdarreola) [inline npc/re/mobs/dungeons/lhz_dun_n.txt]: As per https://forums.warpportal.com/index.php?/topic/207891-kro-warriors-grave-nightmare-biolab-re-adjustment/

Once the mvp is summon a message will appear in map to all players.

Google Translation of the message:
Voice of Envoy: I've come to see a real good-bye. (Not sure of this translation)

**Comment by bruno199** [comment]
@Atemo 
1: no, the chance for drop fragments is same, 5% but have chance drop 2 in each mob
2: yes, normal boss never drop death bin, only mvp

**Comment by lordfelll** [comment]
UP

**Comment by Atemo** [comment]
@lordfelll The pull request has been made to review / discuss the changes!
- Do you have additionnal informations?
- Did you find any bug?
- Do you have any suggestion?

Please contribute !

**Comment by xTakami** [comment]
@Atemo It would be great, if I could finish adding the skills to the monsters.

**Comment by Atemo** [comment]
Officially you don't get a special enchant at 100%, if it's what you mean

**Comment by iubantot** [comment]
> Officially you don't get a special enchant at 100%, if it's what you mean

ohh i see

**Comment by OptimusM** [comment]
@iubantot
Even if u get the enchant that are u looking for, the chance To upgrade that enchant lvl is really low, and could fail, downgrading the current enchant level.
I dont know anyone even on kRo whit lvl 10 special enchant (it’s a really low chance for lvl’s 4 and more)

**Comment by slyx88** [comment]
the mobs do not move but it can cast skills =(
screenrathena000

**Comment by Atemo** [comment]
https://github.com/rathena/rathena/commit/a4fa805469fce962642823ce49357eb7ffd5139b

**Comment by CyberDevil75** [comment]
sorry if you were already know this but the mobs in this dungeon are missing some important items required to complete the quests... it's all described in detail here:
https://github.com/rathena/rathena/issues/3785

**Comment by mazvi** [comment]
how about this?

**Pull request merged**